build/int.cloudbuild.yaml

```diff
@@ -39,6 +39,9 @@ steps:
 - id: apply-dwh
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestDataWarehouse --stage apply --verbose']
+- id: wait-for-post-apply-steps
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['sleep', '$_API_ACTIVATION_SECONDS_WAIT']
```

Comment:
nit: do you want to just pass the arg for the no. of seconds here directly instead of using a variable? Also, `_API_ACTIVATION_SECONDS_WAIT` isn't really accurate.

Reply:
If this wait is needed during local test execution too, doing it within verify func with `time.sleep()` is better.

Reply:
local tests in this still assume running all of the apply stages prior correct?